src/qmapshack/icon/CMapIconSizesSetup.cpp

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************************************
+    Copyright (C) 2014 Oliver Eichler <oliver.eichler@gmx.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#include "icon/CMapIconSizesSetup.h"
+#include "helpers/CSettings.h"
+#include "helpers/CWptIconManager.h"
+#include "poi/IPoiFile.h"
+
+#include <QPushButton>
+#include <QSlider>
+
+CMapIconSizesSetup::CMapIconSizesSetup(QWidget* parent) : QDialog(parent) {
+  setupUi(this);
+
+  SETTINGS;
+  slider_1->valueChanged(cfg.value("Icons/sizeWpt", 22).toInt());
+  slider_2->valueChanged(cfg.value("Icons/sizePoi", 22).toInt());
```

Comment:
I always try to use readable names like sliderWpt or sliderPoi

And valueChanged is  a signal. Either use emit or this is the wrong API to use.